plugins/discovery-gce/src/test/java/org/elasticsearch/discovery/gce/GceDiscoveryTests.java

```diff
@@ -254,4 +254,17 @@ public void testIllegalSettingsMissingZone() {
             assertThat(expected.getMessage(), containsString("one or more gce discovery settings are missing."));
         }
     }
+
+    /**
+     * For issue https://github.com/elastic/elasticsearch/issues/16967
```

Comment:
Can we have a small note on what this test does? link is good but description is better.

Same thing with method name: `testNoRegionReturnsEmptyList` or something like that